xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -901,13 +901,13 @@ int CPVRGUIInfo::GetDuration(void) const
 int CPVRGUIInfo::GetStartTime(void) const
 {
   CSingleLock lock(m_critSection);
-  if (m_playingEpgTag)
+  if (m_playingEpgTag || m_iTimeshiftStartTime)
```

Comment:
Well, maybe not exactly **differently** by means of this code change here beeing wrong, but maybe there currently is some check using gui info to decide whether seek ff/rew skip +/-and that needs to be addressed **additionally** (in anotehr PR). No idea. Have to check if I find the time.